armi/reactor/tests/test_blocks.py

```diff
@@ -1797,6 +1798,45 @@ def test_getReactionRates(self):
         )
 
 
+class BlockInputHeightsTests(unittest.TestCase):
+    @staticmethod
+    def _getAssemblyWithType(assems, nameInBP: str):
+        for assem in filter(lambda a: a.getType() == nameInBP, assems):
+            return assem
+        raise KeyError(nameInBP)
+
+    def test_foundReactor(self):
+        """Test the input height is pullable from blueprints."""
+        r = loadTestReactor()[1]
+        msg = "Input height from blueprints differs. Did a blueprint get updated and not this test?"
+
+        # Grab a block from an assembly, so long as we have the height
+        assem = self._getAssemblyWithType(r.core, "igniter fuel")
```

Comment:
ARMI already provides:

```python
r.core.getFirstAssembly(Flags.IGNITER)
```
or perhaps:

```python
r.core.getFirstAssembly(Flags.IGNITER | Flags.FUEL)
```

It _feels_ to me like that would be simpler than added a whole helper method to this class that does the same thing.

Unless I'm missing something, and that didn't work in this case?

Reply:
I think I added this helper because some of the other methods didn't quite work but I don't think I tried this one. It works so I'll roll it in